src/commands/dsyms/utils.ts

```diff
@@ -0,0 +1,127 @@
+import {exec} from 'child_process'
+import {mkdirSync} from 'fs'
+import glob from 'glob'
+import path from 'path'
+import {promisify} from 'util'
+import {buildPath} from '../../helpers/utils'
+import {Payload} from './interfaces'
+import {checkNonEmptyFile} from './validation'
```

Comment:
Why a different file for this one?

Reply:
i took `validation.ts` from `sourcemaps/validation.ts`
`checkNonEmptyFile` can be moved to `utils.ts` 👍